atlasdb-api/src/main/java/com/palantir/atlasdb/util/MetricsManager.java

```diff
@@ -193,6 +189,12 @@ private synchronized Meter registerOrGetMeter(String fullyQualifiedMeterName) {
         return meter;
     }
 
+    public synchronized Meter registerOrGetTaggedMeter(Class clazz, String metricName, Map<String, String> tags) {
+        MetricName name = getTaggedMetricName(clazz, metricName, tags);
+        // TODO(tboam): do we need to keep a seprate record of registeredTaggedMetrics so we can clean those up too?
```

Comment:
`deregisterMetrics` removes untagged metrics but I don't see the same for tagged metrics.  does this need to be added?